packages/amplify-e2e-core/package.json

```diff
@@ -40,8 +44,5 @@
     "@aws-sdk/client-ec2": "^3.226.0",
     "@aws-sdk/client-rds": "^3.226.0",
     "knex": "~2.4.0"
-  },
-  "peerDependencies": {
-    "@aws-amplify/amplify-cli-core": "4.0.4"
```

Comment:
I see that you're still using `stateManager` etc from this dep, don't you want to keep it?